packages/theming/theme-types/src/Theme.types.ts

```diff
@@ -11,21 +11,31 @@ type SpacingValue = number | string;
 export interface Spacing {
   s2: string;
   s1: string;
+  m?: string;
   // m is now defined as a v2 type
   l1: string;
   l2: string;
   // v2 types
-  none?: SpacingValue;
-  xxxs?: SpacingValue;
-  xxs?: SpacingValue;
-  xs?: SpacingValue;
-  s?: SpacingValue;
-  m?: SpacingValue;
-  l?: SpacingValue;
-  xl?: SpacingValue;
-  xxl?: SpacingValue;
-  xxxl?: SpacingValue;
-  xxxxl?: SpacingValue;
+  sizeNone?: SpacingValue;
```

Comment:
If we want to encourage users to use global tokens for sizing, we shouldn't be adding them to the `Theme` definition.

Reply:
Reverted the new addition.  
Also as mentioned in the description, we have to take the removal of spacing from theme as separate PR which should focus on entirely removal of spacing from theming.